pkg/minikube/cluster/cluster.go

```diff
@@ -584,7 +584,7 @@ func GetServiceURLs(api libmachine.API, namespace string, t *template.Template)
 }
 
 func GetCA(h sshAble) (string, error) {
```

Comment:
I know, I thought about this as well. It is not used right now, but it actually works and I thought it could be useful at some stage. But I guess when we have the need for it, we can get it back. ATM, I am also not seeing the exact use case where we need it. 

I'll remove it